uniprot-config/src/main/resources/return-fields-config/uniref-return-fields.json

```diff
@@ -108,7 +108,7 @@
     "itemType": "SINGLE",
     "name": "sequence",
     "label": "Reference sequence",
-    "paths": ["representativeMember.sequence.value","sequence"],
+    "paths": ["representativeMember.sequence.value"],
     "id": "miscellaneous/sequence",
     "includeInSwagger": true
   },
```

Comment:
maybe i'm wrong, but i think we need to update the entry filtering paths? in uniparc-search-fields.json because the paths will have changed